Assets/MixedRealityToolkit/Utilities/InspectorFields/InspectorGenericFields.cs

```diff
@@ -31,7 +31,10 @@ public static void LoadSettings(T target, List<InspectorPropertySetting> setting
                 foreach (var attr in attrs)
                 {
                     object value = InspectorField.GetSettingValue(settings, propInfo.Name);
-                    propInfo.SetValue(target, value);
+                    if (value != null)
```

Comment:
what happens in the null case? Does a default value fill in?

Reply:
afais it's just taking whatever default is defined in the class - but i'll check again if that's really the case

Reply:
yeah confirmed. it's getting the defaults from the class and this is just overwriting whatever default value is already in there